opm/autodiff/NonlinearSolver_impl.hpp

```diff
@@ -137,7 +137,11 @@ namespace Opm
             }
             converged = report.converged;
             linIters += report.linear_iterations;
-            wellIters += report.well_iterations;
+            if (report.well_iterations != std::numeric_limits<int>::min()) {
+                wellIters += report.well_iterations;
+            } else {
+                wellIters = report.well_iterations;
```

Comment:
Are you sure about this?  Isn't this branch (i.e., the `else` branch) exactly equal to
```C++
wellIters = std::numeric_limits<int>::min();
```
?  Won't that lead to problems elsewhere?